File: js/14-cryptography/03-RSA/GaloisField/GaloisField.js
var groupList = function (ra, op, rb, p) {
  console.log('============================')
  for (let a = ra.min; a <= ra.max; a++) {
    console.log()
    for (let b = rb.min; b <= rb.max; b++) {
      let ab = (op==='*')?a*b:a+b
      ab = ab % p
      console.log(a, op, b, '=', ab, 'mod', p)
    }
  }
}

let ra = {min: 0, max: 6}
groupList(ra, '+', ra, 7)
groupList(ra, '*', ra, 7)

File: py/14-cryptography/GaloisField/GaloisField.py
def groupList(ra, op, rb, p) :
  print('============================')
  for ( a = ra.min  a <= ra.max  a++) :
    print()
    for ( b = rb.min  b <= rb.max  b++) :
       ab = (op==='*')?a*b:a+b
      ab = ab % p
      print(a, op, b, '=', ab, 'mod', p)
    
  


 ra = :min: 0, max: 6
groupList(ra, '+', ra, 7)
groupList(ra, '*', ra, 7)